Sources/ApolloCodegenLib/Templates/SelectionSetTemplate.swift

```diff
@@ -624,7 +624,7 @@ fileprivate extension IR.SelectionSet {
 
 }
 
-fileprivate extension IR.EntityField {
+extension IR.EntityField {
```

Comment:
Considered moving this out to another file since it is being used outside of `SelectionSetTemplate` now however it uses another fileprivate extension within `SelectionSetTemplate` so opted to keep it here for now.

Reply:
If they are being used outside of this file, they should probably be moved into the `Utilities` folder. `fileprivate` is good for it being scope to just the `SelectionSetTemplate`, but if other places are dependent on the behavior of this, we should make that clear by pulling it out and documenting what it's doing more.

Reply:
Pulled these out into a `IR+Formatting.swift` file in the `Extensions` folder since there wasn't a `Utilities` folder in `ApolloCodegenLib`